config/Dockerfiles/JenkinsfileRelease

```diff
@@ -194,6 +197,21 @@ EOF
                             sh "twine upload --config-file /tmp/pypirc -r " + env.PROD_PYPI_REPO + " dist/*"
                         }
                     }
+                    stage('release-container') {
+                        dir('config/Dockerfiles/linchpin'){
+                            string version= sh (
+                                script: "python setup.py --version",
+                                returnStdout: true
+                                ).trim()
+                            )
+                            image = "docker.io/" + env.DOCKER_HUB_USERNAME + "/" + env.DOCKER_HUB_IMAGE
+                            tagged_img = image + ":" + version
+                            sh "buildah bud -t " + image + " ."
+                            sh "buildah push --authfile " + $docker-creds + " " + image
+                            sh "buildah tag " + image + " " + tagged_image
+                            sh "buildah push --authfile " + $docker-creds + " " + image
```

Comment:
why push twice?

Reply:
The first pushes to contrainfra/linchpin:latest and the second pushes to contrainfra/linchpin:${version}